libraries/wallet/include/golos/wallet/wallet.hpp

```diff
@@ -1390,6 +1390,189 @@ namespace golos { namespace wallet {
                 const std::string& from, const std::string& to,
                 const std::string& start_date, const std::string& stop_date, bool broadcast);
 
+            /**
+             * Create worker proposal based on specified post
+             *
+             * @param author author of the post
+             * @param permlink permlink of the post
+             * @param type type of worker proposal
+             * @param broadcast true if you wish to broadcast the transaction
+             */
+            annotated_signed_transaction create_worker_proposal(
+                const std::string& author, const std::string& permlink, worker_proposal_type type, bool broadcast
+                );
+
+            /**
+             * Edit worker proposal based on specified post
+             *
+             * @param author author of the post
+             * @param permlink permlink of the post
+             * @param type new type of worker proposal
+             * @param broadcast true if you wish to broadcast the transaction
+             */
+            annotated_signed_transaction edit_worker_proposal(
+                const std::string& author, const std::string& permlink, worker_proposal_type type, bool broadcast
+                );
```

Comment:
Refactor to one opertation: `create_worker_proposal()` + `edit_worker_proposal()`  -> `worker_proposal()` (like `post()`)